modules/docs/src/main/mdoc/docs/mtl-combinators.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: docs
+title: MTL Combinators
+---
+
+# MTL Combinators
+
+The `cats-retry-mtl` module provides two additional retry methods that operating with errors produced 
+by `ApplicativeHandle` from [cats-mtl](https://github.com/typelevel/cats-mtl).
+
+## Installation
+
+To use `cats-retry-mtl`, add the following dependency to your `build.sbt`:
+```scala mdoc:passthrough
+println(
+  s"""
+  |```
+  |val catsRetryVersion = "${retry.BuildInfo.version.replaceFirst("\\+.*", "")}"
+  |libraryDependencies += "com.github.cb372" %% "cats-retry-mtl" % catsRetryVersion
+  |```
+  |""".stripMargin.trim
+)
+```
+
+## Interaction with MonadError retry
+
+MTL retry works independently from `retry.retryingOnSomeErrors`. The operations `retry.mtl.retryingOnAllErrors` and 
+`retry.mtl.retryingOnSomeErrors` evaluating retry exclusively on errors produced by `ApplicativeHandle`.
+Thus errors produced by `MonadError` are not being taken into account and retry is not triggered.
+
+If you want to retry in case of any error, just chain the methods:  
```

Comment:
Let's avoid words like "just" in documentation.

```suggestion
If you want to retry in case of any error, you can chain the methods:
```

Reply:
Fixed